dtmi/rigado/hobomx1102a-1.json

```diff
@@ -0,0 +1,53 @@
+{
+  "@context": [
+    "dtmi:iotcentral:context;2",
+    "dtmi:dtdl:context;2"
+  ],
+  "@id": "dtmi:rigado:HoboMX1102A;1",
+  "@type": "Interface",
+  "contents": [
+    {
+      "@type": [
+        "Telemetry",
+        "Humidity"
```

Comment:
to use the `percent` unit, you must use `RelativeHumidity`, this telemetry should  look like.

```
{
  "@type": [
    "Telemetry",
    "RelativeHumidity"
  ],
  "displayName": {
    "en": "Relative Humidity"
  },
  "name": "relativeHumidity",
  "schema": "double",
  "unit": "percent"
}
```